### PR TITLE
chore: Remove Go imports for initflag

### PR DESCRIPTION
**Pull request by LKreutzer**
Signed-off-by: Lars Kreutzer <lars.kreutzer@tngtech.com>

## Summary

- The Go imports of the initflag package are removed
- The package is still used by openwrt
  - This usage should be cleaned up at a later point
- The initflag Bazel libraries are removed 
 
## Test Plan

- CI
- Run `cd ${MAGMA_ROOT}/orc8r/gateway/go &&  go test ./...`

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: orc8r/lib/go/initflag/syslog.go [hunk 1]
```diff
@@ -1,161 +0,0 @@
...
-	"flag"
-	"fmt"
-	"log"
-	"log/syslog"
-	"os"
-	"path/filepath"
-	"strings"
-)
-
-const syslogFlag = "stderr2syslog"
```
Comment (crasu) [inline orc8r/lib/go/initflag/syslog.go]: This seems to be used in the magma openwrt deployment here:
./openwrt/gateway/configs/etc/init.d/magma.template

**Review by crasu — CHANGES_REQUESTED** [review]

File: orc8r/lib/go/initflag/syslog.go (hunk 1 above)
Comment (crasu) [reply orc8r/lib/go/initflag/syslog.go]: I do not know if we can remove the openwrt stuff as well.

**Comment by nstng** [comment]
converted to draft - initflag still needed in other places. needs to be adapted

**Comment by LKreutzer** [comment]
Changed the commit to only remove the side-effect imports in the Go code but the package itself since it is still used in the openwrt code.

**Comment by LKreutzer** [comment]
Added commit to also remove the imports in the bazel code.

**Comment by LKreutzer** [comment]
As discussed with @crasu @sebathomas @jheidbrink @alexzurbonsen:

- Add initflag functionality to bazel in https://github.com/magma/magma/pull/13532
- Closing this PR and move refactoring of initflag functionality to future task.

**Pull request closed without merging**